dockers/docker-fpm-frr/frr/supervisord/supervisord.conf.j2

```diff
@@ -29,7 +29,7 @@ stderr_logfile=syslog
 dependent_startup=true
 
 [program:zebra]
-command=/usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -M fpm -M snmp
+command=/usr/lib/frr/zebra -A 127.0.0.1 -s 90000000 -K 300 -r -M fpm -M snmp --v6-rr-semantics
```

Comment:
Could you please help clarify why these changes are necessary on this topic? What will happen without these changes?